semantic_search/schemas.py

```diff
@@ -32,7 +32,7 @@ def normalize_document(cls, v, field):
                 # uids_to_docs expects a list of strings and yields a list of dictionaries. We
                 # convert the generator to a list, and then index its first element, and then
                 # unpack the dictionary and pass its contents as keyword arguments to Document.
-                normalized_docs.append(Document(**list(uids_to_docs([doc]))[0][0]))
+                normalized_docs.append(Document(**list(uids_to_docs([doc]))[0][0])) 
```

Comment:
Looks like there is an extra whitespace. Did you call `black .` from the top-level of the project?

Reply:
Fixed.

Reply:
Oh I might have missed it. I remember calling pytest and mypy. Sorry about that, thanks for fixing.